Project/zepeto-modules/Assets/Zepeto Gesture Module/ZepetoScript/GestureLoader.ts

```diff
@@ -86,29 +88,34 @@ export default class GestureLoader extends ZepetoScriptBehaviour {
         this._myCharacter.CancelGesture()
 
         // check if isGestureLooping is true and it is not a pose
-        if(this.isGestureLooping && this._isRepeatableContentType(type))
-        {            
+        if(this.isGestureLooping && this._isGestureNotAPose(type))
+        {    
             this.gestureLoop = this.StartCoroutine(this.setGestureLoop(animation))
         }
+        //Check if the Gesture is a not pose
+        else if(!this._isGestureNotAPose(type))
+        {
+            //activate the pose
+            this._poseIsRunning = true;
+            this.gestureLoop = this.StartCoroutine(this.setGesturePose(animation))
+        }
         //When the isGestureLooping is false
         else{
-            //If there is another animation running, cancel it
             this._myCharacter.SetGesture(animation)
         }   
     }
 
-
     //This function check if the gesture repeatable and return true, if it's not, it returns false.
-    private _isRepeatableContentType(type: OfficialContentType[])
+    private _isGestureNotAPose(type: OfficialContentType[]):bool
```

Comment:
Gesture is a term that refers to gesture, pose, ... altogether AND it also means the 'gestures' itself too.

Since we now handle playback settings for both gestures and poses, this name can be ambiguous.

What about simply saying `isNotPose`?